### ISSUE TITLE
no sitepage text margins on viewport < 768

### ISSUE THREAD

**Issue by githubertus** [opensource-socialnetwork/opensource-socialnetwork#690]
![sitepage-margins](https://cloud.githubusercontent.com/assets/9904364/12018513/28fb6956-ad64-11e5-950a-859e5a17ad5f.png)

**Comment by ctlui** [comment]
May be solution for this & 689:
https://github.com/opensource-socialnetwork/opensource-socialnetwork/blob/v4.x/components/OssnSitePages/pages/page.php#L19
change to:
``` <div class="col-md-12 col-xs-12 ossn-site-pages-inner  ossn-page-contents">```

add in css:
``` 
@media only screen
and (max-width : 1360px) {
.ossn-site-pages{
    margin-left: 15px;
    margin-right: 15px
    }
}
@media only screen
and (min-width : 1360px) {
.ossn-site-pages{
    margin-left: 80px;
    margin-right: 80px
    }
}	
```

**Comment by githubertus** [comment]
fixed #735 #736
